docs/ibexa_products/editions.md

```diff
@@ -57,4 +57,13 @@ Compare all features available in [[= product_name_headless =]], [[= product_nam
 | [Storefront](storefront.md) |  |  | &#10004; |
 | [Transactional emails](transactional_emails.md) |  |  | &#10004; |
 | [Discounts](discounts.md) | | | &#10004; |
-| [Symbol attribute type](symbol_attribute_type.md) | &#10004; | &#10004; | &#10004; |
\ No newline at end of file
+| [Symbol attribute type](symbol_attribute_type.md) | &#10004; | &#10004; | &#10004; |
```

Comment:
I know it's out of scope for this PR, but could you please move Symbol below Date and time attribute type?

Reasoning:
1) It makes sense to group PIM together
2) Right now the lower you in the table the more exclusive the features are - with Symbol being the exception